test/browser/spec/ol/Map.test.js

```diff
@@ -27,19 +29,22 @@ import VectorLayer from '../../../../src/ol/layer/Vector.js';
 import VectorTileLayer from '../../../../src/ol/layer/VectorTile.js';
 import WebGLVectorLayer from '../../../../src/ol/layer/WebGLVector.js';
 import {tile as tileStrategy} from '../../../../src/ol/loadingstrategy.js';
+
```

Comment:
Hmm. These blank lines shouldn't be here. I'll see if the codemod needs an update.

Reply:
I'm feeling like this may be a jscodeshift artifact that we might not have control over. I'll manually delete these two blank lines for now and will dig further if it looks like a bigger problem later.